docs/user/filesystem.md

```diff
@@ -314,6 +314,11 @@ files will use the page cache.  This default value for this is "off".
 With no options specified attr and dentry timeouts will be 1 second, dentry-dir
 and ndentry timeouts will be 5 seconds, and data caching will be enabled.
 
+Readir caching will be enabled when the dfuse-dentry-time setting is non-zero and when supported by
+libfuse however on many distributions the system libfuse is not able to support this feature.
+Libfuse version 3.5.0 is required at both compile and run-time.  Use `dfuse --version` or the
+runtime logs to see the fuse version used and if the feature is compiled into dfuse.
```

Comment:
would be helpful to add the line that corresponds in the runtime logs to look for

Reply:
oh or maybe it's just that new info log you added.
i recall there was a parameter printed that i had to check